README.md

```diff
@@ -85,9 +85,11 @@ The suite can then be launched from the login node with:
 ```shell
 git clone https://github.com/charmed-hpc/charmed-hpc-benchmarks
 cd charmed-hpc-benchmarks
-reframe -C config/azure_config.py -c checks -r -R
+reframe -C config/azure_config.py -c checks -r -R -S slurmrestd_api_check.slurmrestd_hostname=juju-50c1e0-2
```

Comment:
[nit]: Is it possible to use the long version of the flags here? Reason saying is that it's generally better to be [verbose and descriptive in shell scripts](https://clig.dev/#arguments-and-flags) because then your users don't need to look up the meaning of short-hand flags.

For example, anyone that isn't familiar with ReFrame or use it every day will likely have no idea what these flags mean unless they scour ReFrame's documentation. It's a bit more typing for us, but it helps make things more clear for project newcomers.